pyscf/pbc/cc/eom_kccsd_ghf.py

```diff
@@ -1279,6 +1280,551 @@ def make_imds(self, eris=None):
         imds.make_t3p2_ea(self._cc)
         return imds
 
+########################################
+# EOM-EE-CCSD
+########################################
+
+def kernel_ee(eom, nroots=1, koopmans=False, guess=None, left=False,
+           eris=None, imds=None, partition=None, kptlist=None,
+           dtype=None, **kwargs):
+    '''See `kernel()` for a description of arguments.
+
+    This method is merely a simplified version of kernel() with a few parts
+    removed, such as those involving `eom.mask_frozen()`. Slowly they will be
+    added back for the completion of program.
+    '''
+    cput0 = (time.clock(), time.time())
+    log = logger.Logger(eom.stdout, eom.verbose)
+    if eom.verbose >= logger.WARN:
+        eom.check_sanity()
+    eom.dump_flags()
+
+    if imds is None:
+        imds = eom.make_imds(eris=eris)
+
+    nkpts = eom.nkpts
+
+    if kptlist is None:
+        kptlist = range(nkpts)
+
+    # TODO mask frozen-orbital indices
+
+    if dtype is None:
+        dtype = np.result_type(*imds.t1)
+
+    # evals = np.zeros((len(kptlist),nroots), np.float)
+    # evecs = np.zeros((len(kptlist),nroots,size), dtype)
+    # convs = np.zeros((len(kptlist),nroots), dtype)
+    evals = [None]*len(kptlist)
+    evecs = [None]*len(kptlist)
+    convs = [None]*len(kptlist)
+
+    for k, kshift in enumerate(kptlist):
+        print("\nkshift =", kshift)
+        # vector size and thus, nroots depend on kshift in the case of even nkpts,
+        size = eom.vector_size(kshift)
+        nroots = min(nroots, size)
+
+        matvec, diag = eom.gen_matvec(kshift, imds, left=left, **kwargs)
+        # TODO update `diag` in case of frozen orbitals
+
+        # TODO allow user provided guess vector
+        # if guess:
+        #     user_guess = True
+        #     assert len(guess) == nroots
+        #     for g in guess:
+        #         assert g.size == size
+        # else:
+        #     user_guess = False
+        #     guess = eom.get_init_guess(kshift, nroots, koopmans, diag)
+        user_guess = False
```

Comment:
Add explanations on why `user_guess` is always `False`, and why `guess` from previous `kshift` cannot be used.